configs/hugetlb.go

```diff
@@ -0,0 +1,9 @@
+package configs
+
+type Hugetlb struct {
+	// which type of hugepage to limit.
+	Pagesize string `json:"page_size"`
```

Comment:
Hmmm this type of stuff makes it a very hard API to consume. How would I know what values are allowed? How to I know which are available?

I'm OK starting with this, but can we at least put example of valid values? :)